<!DOCTYPE html>
<html lang="en">
<head>

<title>CCGLab</title>

<!--

Template 2099 Scenic

http://www.tooplate.com/view/2099-scenic

-->

<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<meta name="description" content="">
<meta name="keywords" content="">
<meta name="author" content="">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

<link rel="stylesheet" href="css/bootstrap.min.css">
<link rel="stylesheet" href="css/font-awesome.min.css">
<link rel="stylesheet" href="css/magnific-popup.css">

<link rel="stylesheet" href="css/owl.theme.css">
<link rel="stylesheet" href="css/owl.carousel.css">

<!-- MAIN CSS -->
<link rel="stylesheet" href="css/tooplate-style.css">

</head>
<body>

<!-- PRE LOADER -->
<div class="preloader">
     <div class="spinner">
          <span class="sk-inner-circle"></span>
     </div>
</div>


<!-- MENU -->
<div class="navbar custom-navbar navbar-fixed-top" role="navigation">
     <div class="container">

          <!-- NAVBAR HEADER -->
          <div class="navbar-header">
               <button class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                    <span class="icon icon-bar"></span>
                    <span class="icon icon-bar"></span>
                    <span class="icon icon-bar"></span>
               </button>
               <!-- lOGO -->
               <a href="index.html" class="navbar-brand">CCG Lab</a>
          </div>

          <!-- MENU LINKS -->
          <div class="collapse navbar-collapse">
               <ul class="nav navbar-nav navbar-right">
                    <li><a href="#home" class="smoothScroll">Home</a></li>
                    <li><a href="#about" class="smoothScroll">Research</a></li>
                    <li><a href="#publication" class="smoothScroll">Publication</a></li>
                    <li><a href="#people" class="smoothScroll">People</a></li>
                    <li><a href="#joinus" class="smoothScroll">Join</a></li>
                    <li><a href="#contact" class="smoothScroll">Contact</a></li>
               </ul>
          </div>

     </div>
</div>


<!-- HOME -->
<section id="home" class="parallax-section">
     <div class="overlay"></div>
     <div class="container">
          <div class="row">

               <div class="col-md-8 col-sm-12">
                    <div class="home-text">
                         <h1>Welcome to computational cancer genomics lab!</h1>
                         <p> We write code to advance precision oncology</p>
                         <ul class="section-btn">
                              <a href="#about" class="smoothScroll"><span data-hover="Discover More">Discover More</span></a>
                         </ul>
                    </div>
               </div>

          </div>
     </div>

     <!-- Video -->
     
       <!-- video src="https://user-images.githubusercontent.com/88851635/129434805-97c2c130-1ef7-4071-ae51-820802a7e668.mp4" data-canonical-src="https://user-images.githubusercontent.com/88851635/129434805-97c2c130-1ef7-4071-ae51-820802a7e668.mp4" -->
       <video src="https://user-images.githubusercontent.com/169707/126715420-991ad821-9ac8-4b66-b79e-e0966e0f3a89.mp4" data-canonical-src="https://user-images.githubusercontent.com/169707/126715420-991ad821-9ac8-4b66-b79e-e0966e0f3a89.mp4" controls="controls" muted="muted" style="max-height:640px;">
     </video>
     </section>


<!-- ABOUT -->
<section id="about" class="parallax-section">
     <div class="container">
          <div class="row">

               <div class="col-md-offset-2 col-md-10 col-sm-12" -->
               <!-- div class="col-md-10 col-sm-12" -->
                    <div class="about-info">
                      <h1> <b>Research<b></h1>
                      <br>
                      <br>
                      <p style="text-align:justify"><font size="5"><b>Welcome to the Computational Cancer Genomics Lab at the <a href="http://www.uhnresearch.ca/institutes/pm">Princess Margaret Cancer Center</a> at the University of Toronto. We study role of genomic variations in driving tumor progression.</b></font></p>
                      <br>
                      <p style="text-align:justify"><font size="4"><b>Our lab studies cancer biology by building computational tools with state-of-the art genomics, biophysics, and machine learning techniques. We are looking for enthusiastic researchers interested in omputational biology to <a href="#joinus" class="smoothScroll"><b>JOIN US</a>!</b></font></p>
                      <br>
                      <p style="text-align:justify"><font size="4"> Rapid declines in sequencing costs have enabled large-scale genome and exome sequencing for various cancer cohorts. A critical shared objective among such studies has been to understand how genomic variants affect tumor etiology. How may we develop robust quantitative models to predict the impact of somatic mutations on gene expression and protein function? Furthermore, how may we leverage these quantitative models to prioritize genomic variants and utilize this knowledge to develop new cancer therapeutics? My lab is interested in developing integrative methods that use multiple data resources and cross-disciplinary approaches to address questions of this nature. </font> </p>
                      <br>
                      <br>
                      <img src="braf.transparent.png" align="left" width="300" height="250" HSPACE="20">
                      <p style="text-align:justify"><font size="4"> Exome sequencing remains the primary sequencing platform in cancer studies, owing mainly to its relatively lower costs and strong ability to identify clinically actionable variants. Among coding variants, quantifying the molecular impact and prioritization of missense mutations in cancer cohorts remains challenging. Prior efforts to quantify the impact of missense mutations relied on statistical frameworks that evaluate the positive selection signal for somatic mutations. However, these approaches fail to provide mechanistic insights into how high impact missense mutations drive tumor progression. In our previous work, we have developed methods that integrate protein structure and protein motion information to evaluate molecular impact of cancer mutations and identify cancer driver genes. As a continuation of these works, my lab will develop new integrative machine learning methods integrating protein structure and cancer mutations to identify novel drug targets and predict efficacy & side effects of drugs among cancer patients. </font> </p>
                      <br>
                      <br>

                      <img src="paperE_schematic.jpeg" align="right" width="300" height="350" HSPACE="20">
                      <p style="text-align:justify"><font size="4"> The overwhelming majority of cancer mutations fall within non-coding regions of the genome. Despite their higher frequency, clear insights into how non-coding mutations play causal roles in various cancer types remain limited. One of my lab’s primary research goals is to build methodologies for assessing the molecular impact of variants, as well as prioritizing non-coding mutations and SVs in different cancer types. Precision medicine initiatives in oncology, such as the cancer moonshot aim to bring better diagnosis and treatment efficacy. Currently, genomics and transcriptomics-based technologies constitute the core of precision medicine in cancer. The majority of these efforts involve generating comprehensive catalogs of driver alterations. Such catalogs have translated into a few instances in which targeting actionable genes leads to therapeutic developments in cancer.
                      </font> </p>
                      <br>
                      <br>

                      <img src="SVFX_schematic_updated.png" align="left" width="300" height="350" HSPACE="20">
                      <p style="text-align:justify"><font size="4"> The overwhelming majority of cancer mutations fall within non-coding regions of the genome. Despite their higher frequency, clear insights into how non-coding mutations play causal roles in various cancer types remain limited. Similarly, current genomic studies primarily focus on understanding the effects of point mutations and INDELs on phenotype while overlooking those of genomic structural variations (SVs), despite the fact that SVs are more likely to affect gene expression and play critical roles in various cancer types. As with non-coding mutations, our understanding of how SVs influence cancer progression remains limited. One of my lab’s primary research goals is to build methodologies for assessing the molecular impact of variants, as well as prioritizing non-coding mutations and SVs in different cancer types.
                      </font> </p>
                      <br>
               </div>

          </div>
     </div>

</div>

</section>


<!-- ABOUT -->
<section id="publication" class="parallax-section">
     <div class="container">
          <div class="row">

               <div class="col-md-offset-2 col-md-10 col-sm-12" -->
               <!-- div class="col-md-10 col-sm-12" -->
                    <div class="publication-info">
                      <h1> <b> Recent Publications </b> </h1>
                      <br>
                      <p style="text-align:left"><font size="3"><b><a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00879-x">Whole-genome sequencing of phenotypically distinct inflammatory breast cancers reveals similar genomic alterations to the more commonplace non-inflammatory breast cancers</a>,</b>
                      Xiaotong Li; <u>Sushant Kumar</u>; Arif Harmanci; …. Naoto T. Ueno; Savitri Krishnamurthy; Lajos Pusztai; and Mark Gerstein. <b>Genome Medicine (2021)</b></font></p>

                      <p style="text-align:left"><font size="3"><b><a href="https://science.sciencemag.org/content/372/6537/eabf7117.long">Haplotype-resolved diverse human genomes and integrated analysis of structural variation</a>,</b>
                      Peter Ebert; Peter Audano; Qihui Zhu; Bernardo Rodriguez-Martin; …. <u>Sushant Kumar</u>; …... Tobias Marschall; and Evan E. Eichler <b>Science (2021)</b></font></p>

                      <p style="text-align:left"><font size="3"><b><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02178-x">SVFX: a machine learning framework to quantify the pathogenicity of structural variants</a>,</b>
                      <u>Sushant Kumar</u>*; Arif Harmanci*; Jagath Vytheeswaran; and Mark Gerstein <b>Genome Biology (2020)</b></font></p>

                      <p style="text-align:left"><font size="3"><b><a href="https://www.sciencedirect.com/science/article/pii/S0092867420301136">Passenger Mutations in More Than 2,500 Cancer Genomes: Overall Molecular Functional Impact and Consequences</a>,</b>
                      <u>Sushant Kumar</u>*; Jonathan Warrell*; Shantao Li; Patrick McGillivray; William Meyerson; Leonidas Salichos;….. Ekta Khurana; and Mark Gerstein.  <b>Cell (2020)</b></font></p>

                      <p style="text-align:left"><font size="3"><b><a href="https://www.nature.com/articles/s41586-020-1965-x">Analyses of non-coding somatic drivers in 2,658 cancer whole genomes</a>,</b>
                      Esther Rhienbay; Morten Nielsen; Federico Abascal; Jermiah Wala;….. <u>Sushant Kumar</u>; …... Jakob Pedersen; and Gad Getz.  <b>Nature (2020)</b></font></p>

                      <p style="text-align:left"><font size="3"><b><a href="https://www.pnas.org/content/116/38/18962.long">Leveraging protein dynamics to identify cancer mutational hotspots in 3D-structures</a>,</b>
                      <u>Sushant Kumar</u>; Declan Clarke; and Mark Gerstein.  <b>PNAS (2019)</b></font></p>

                      <br>
                      <br>
                      <br>
                      <p style="text-align:left"><font size="4"><b>For full publication list please check our <a href="https://scholar.google.com/citations?user=ujvDsFUAAAAJ&hl=en">google scholar page</a></b></font></p>
               </div>

          </div>
     </div>
  </div>
</section>


<!-- PEOPLE -->
<section id="people" class="parallax-section">
     <div class="container">
          <div class="row">

               <div class="col-md-offset-2 col-md-10 col-sm-12" -->
               <!-- div class="col-md-10 col-sm-12" -->
                    <div class="people-info">
                    <h1> <b> Team </b></h1>
                    <br>
                    <br>
                     <img src="IMG_SK.jpg" align="left" width="150" height="200" HSPACE="20">
                     <br>
                      <p style="text-align:justify"><font size="4">Sushant Kumar is an Assistant Professor in the <b><a href="https://medbio.utoronto.ca/faculty/kumar">Department of Medical Biophysics</b></a> at the University of Toronto. He is also a scientist at the <b><a href="http://www.uhnresearch.ca/institutes/pm">Princess Margaret Cancer Center</a></b>.
                      His expertise and research interests are in computational biology/bioinformatics, cancer biology, genomics, machine learning, and biophyics.</font></p>
                     <br>
                     <br>
                     <br>
                     <br>
                     <br>
                     <br>
                     <br>
                     <br>
                     <img src="pngwing.com.png" align="left" width="150" height="200" HSPACE="20">
                     <br>
                     <br>
                     <br>
                     <br>
                     <br>
                     <p style="text-align:justify"><font size="4">We are looking for enthusiastic researchers interested in computational biology and cancer research to <a href="pai-lab.org/#join">JOIN US</a>!</b></font></p>

               </div>

          </div>
     </div>

</div>
</section>



<!-- PEOPLE -->
<section id="joinus" class="parallax-section">
     <div class="container">
          <div class="row">

               <div class="col-md-offset-2 col-md-10 col-sm-12" -->
               <!-- div class="col-md-10 col-sm-12" -->
                    <div class="joinus-info">
                    <h1><b>Join</b></h1>
                    <br>
                    <br>
                     <img src="torontodowntown.jpeg" align="left" width="950" height="300" VSPACE=40>
                     <br>
                     <br>
                     <br>
                     <p style="text-align:justify" style="color:blue;"><font size="5"><u> Postdoctoral Associate </u></font></p>
                      <p style="text-align:justify"><font size="4"><b>Job description:</b> Applications are invited for postdoctoral positions in the computational cancer genomics laboratory at the Princess Margaret Cancer Center. We are interested to work with highly motivated person interested in building tools and analyzing cancer genome using long-read, single-cell and cell free DNA techonologie. They will have opportunities to collaborate with a diverse group of experimental and computational biologists at the Princess Margaret Cancer Center. We are looking for candidates with prior experience in computational biology.</font></p>
                     <br>
                     <p style="text-align:justify"><font size="4"><b>Eligibility requirement:</b> A PhD in computational biology, bioinformatics, genomics, computer science or a related field is required. A strong computational background, proficiency in at least one programming language, machine learning, and knowledge of statistics are also required. </font></p>
                     <br>
                     <p style="text-align:justify"><font size="4"><b>How to Apply:</b> Interested candidates should send a cover letter, CV and names of three references to Dr. Kumar (sushantDotkumarAtuhnresearch.com).
                     <br>
                     <br>
                     <p style="text-align:justify" style="color:blue;"><font size="5"><u> Graduate and Undergraduate Students </u></font></p>
                     <p style="text-align:justify"><font size="4">Interested students should send an email to Dr. Kumar (sushantDotkumarAtuhnresearch.com) to discuss potential projects in the lab.

               </div>

          </div>
     </div>

</div>
</section>


<!-- FOOTER -->
<footer>
     <div class="container">
          <div class="row">



               <div class="clearfix"></div>

               <div class="col-md-12 col-sm-12">
                    <div class="copyright-text">
                         <p>Copyright © 2021 Sushant Kumar

                         | Design: Tooplate</p>
                    </div>
               </div>

          </div>
     </div>
</footer>

<!-- SCRIPTS -->
<script src="js/jquery.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/jquery.parallax.js"></script>
<script src="js/owl.carousel.min.js"></script>
<script src="js/jquery.magnific-popup.min.js"></script>
<script src="js/magnific-popup-options.js"></script>
<script src="js/modernizr.custom.js"></script>
<script src="js/smoothscroll.js"></script>
<script src="js/custom.js"></script>

</body>
</html>
